views/metrics/charts/quality_pie.html

```
@@ -0,0 +1,49 @@
+{{if .ChartData}}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)